assets/built-in/transformers/dockerfilegenerator/mappings/nodeversions.yaml

```diff
@@ -4,7 +4,11 @@ metadata:
   name: NodeVersionsMapping
 spec:
   nodeVersions:
-    - "v16.16.0"
-    - "v14.20.0"
-    - "v12.22.12"
-    - "v10.24.1"
+    "v16.16.0":
```

Comment:
Putting it as a dictionary voids the order. That's why we might want to use a list.

Reply:
we are sorting it so it doesn't matter